components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -261,12 +261,12 @@ export default function () {
                 )}
             </div>
             <div>
-                <h3 className="mt-12">Delete Project</h3>
+                <h3 className="mt-12">Remove Project</h3>
                 <p className="text-base text-gray-500 dark:text-gray-400 pb-4">
-                    Removing the project from this team will also remove team members access to it.
+                    Removing this project will also remove all associated data with this project, including workspaces.
```

Comment:
**thought**: While we don't delete data like workspaces and prebuilds which we could consider doing in the future, we do remove project-level environments variables, right? See relevant design from https://github.com/gitpod-io/gitpod/issues/7273#issuecomment-1096680115:

We could :a: open a follow-up issue to discuss data deletion and :b: rephrase this to something like the following:

```suggestion
                    This will delete the project and all project-level environment variables you've set for this project.
```

Reply:
Would it be worthwhile to keep this a bit more generic and future-proof, something like:

> This will delete the project and all project-level data you've set for this project.

... or is that too vague and better that we're explicit? I assume we'd have to revisit this wording if we adjust what we delete related to a project being removed over time.

Reply:
Going with your suggestion to just be more clear for now with what happens.